### PR TITLE
Small refactor

### PR DESCRIPTION
**Pull request by Alex6323**

### CODE REVIEW

File: docker/assets/grafana/dashboards/analytics_dashboard.json [hunk 1]
```diff
@@ -3209,7 +3191,7 @@
           "tags": []
         }
       ],
-      "title": "Consumed Output Distribution",
+      "title": "Number of Transactions per Number of Consumed Outputs /${aggregation_interval}",
```
Comment (Alexandcoats) [inline docker/assets/grafana/dashboards/analytics_dashboard.json:3194]: Overly verbose. Prefer the original title.

Comment (Alex6323) [reply docker/assets/grafana/dashboards/analytics_dashboard.json:3194]: Fair enough, but I rather have the title verbose than the legend and the popup overlay. This title makes it clear as well what the numbers stand for while the original proposal had yo make this clear individually, which resulted in more verbosity than this solution. Also ... I wouldnt have picked it myself, but we have a lot of "Number of ..." in the other panels so I wanted it to be consistent.

Comment (Alexandcoats) [reply docker/assets/grafana/dashboards/analytics_dashboard.json:3194]: I don't think it makes it clear at all.

Comment (Alex6323) [reply docker/assets/grafana/dashboards/analytics_dashboard.json:3194]: what's unclear?

Comment (Alexandcoats) [reply docker/assets/grafana/dashboards/analytics_dashboard.json:3194]: It simply does not describe what the stat is.

**Review by Alexandcoats — APPROVED** [review]

**Pull request merged**